docs/version0.37.md

```diff
@@ -75,6 +76,26 @@ A new message, `JVMPORT049I`, is added to help find the system dump files for `k
 
 For more information about system dumps and piped system dumps, see [System dumps on Linux](xdump.md#system-dumps-on-linux).
 
+### ![Start of content that applies to Java 19 plus](cr/java19plus.png) New JDK 19 features
+
+The following features are supported by OpenJ9:
+
+- [JEP 424](https://openjdk.java.net/jeps/424): Foreign Function & Memory API (Preview)
+- [JEP 425](https://openjdk.java.net/jeps/425): Virtual Threads (Preview)
+- [JEP 426](https://openjdk.java.net/jeps/426): Vector API (Fourth Incubator)
+- [JEP 428](https://openjdk.java.net/jeps/428): Structured Concurrency (Incubator)
+    - This is applicable but not implemented
+
+The following features are implemented in OpenJDK and available in any build of OpenJDK 19 with OpenJ9:
+
+- [JEP 405](https://openjdk.java.net/jeps/405): Record Patterns (Preview)
+- [JEP 422](https://openjdk.java.net/jeps/422): Linux/RISC-V Port
```

Comment:
I don't think this should be mentioned. It's from jdk19 anyway. The implementation in OpenJDK is not applicable to OpenJ9.

Reply:
Ok. Pls mention it under a different heading that indicates OpenJ9 doesn't support RISC-V at this time.

Reply:
@Sreekala-Gopakumar if it's not applicable to OpenJ9, we shouldn't be listing it. The topic already says that any OpenJDK features that aren't listed aren't applicable to OpenJ9. 
Please keep updates like this separate - this issue is for updating the o/s list for 0.39, it shouldn't contain fixes to 0.37 docs. It's confusing and would create problems if you had to double-fix something or remove an update.

Reply:
Point noted, Esther. Thanks!